client/js/context/UserContext/UserProvider.js

```diff
@@ -6,9 +6,8 @@ class UserProvider extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      token: "Test Token",
-      token: "",
-      id: ""
+      token: null,
+      id: null
     };
   }
 
```

Comment:
we should not make a component lifecycle method with async await. Please remove async await.